src/Xamarin.Android.Build.Tasks/Xamarin.Android.Build.Tasks.csproj

```diff
@@ -106,6 +106,9 @@
     <Reference Include="Xamarin.Build.AsyncTask">
       <HintPath>..\..\packages\Xamarin.Build.AsyncTask.0.3.4\lib\netstandard2.0\Xamarin.Build.AsyncTask.dll</HintPath>
     </Reference>
+    <Reference Include="pdb2mdb.exe">
+      <HintPath>..\..\bin\$(Configuration)\lib\xamarin.android\xbuild\Xamarin\Android\pdb2mdb.exe</HintPath>
+    </Reference>
```

Comment:
@dellis1972 should we ILRepack this exe? Or ship it in the installer?

Reply:
*I* am not opposed to shipping this in the installer, as it might be useful for one-off testing & diagnostics.

Reply:
If we ship it , will that .exe work with .net or does it rely on mono functionality? Also does it need a specific Cecil/Cecil.Mdb?